functions/helper_lib.sh

```diff
@@ -112,12 +112,22 @@ get_docker_configuration_file() {
   CONFIG_FILE='/dev/null'
 }
 
+if command -v jq &> /dev/null; then
+  HAVE_JQ=true
+else
+  HAVE_JQ=false
+fi
+
 get_docker_configuration_file_args() {
   OPTION="$1"
 
   get_docker_configuration_file
 
-  grep "$OPTION" "$CONFIG_FILE" | sed 's/.*://g' | tr -d '" ',
+  if "$HAVE_JQ"; then
```

Comment:
Why `$HAVE_JQ`  and not just `if command -v jq`?

Reply:
Two reasons.

1. Since the main script calls `. ./functions/helper_lib.sh`, in its current form, `command` will be needed only once, versus potentially many times if it is called each time the `get_docker_configuration_file_args` function is executed
2. Future extensibility - if anywhere else in `helper_lib.sh` wants to use jq in the future, they can just use the already-evaluated `HAVE_JQ`

Reply:
Makes sense, but perhaps move it into `req_programs()` so we bunch all those things into one function.